ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/XARecoveryModule.java

```diff
@@ -293,6 +295,11 @@ public String id() {
         return "XARecoveryModule:" + _recoveryManagerClass;
     }
 
+    @Override
+    public boolean hasWorkLeftToDo() {
```

Comment:
Please can I request some tests similar to https://github.com/jbosstm/narayana/blob/main/ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/recovery/RecoverySuspendTest.java to be added for the XARecoveryModule work left to do checking?

Reply:
Yep sure, I was already working on testing but I wanted to start a `Draft` PR ASAP. Thanks for asking :smile:

Reply:
Thank you